Scripts/writeDataFromApiToFiles/index.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const fetch = require('node-fetch');
+const {FileSystemWriter} = require('../../Classes/FileSystemJSON');
+
+const REQUESTS_PER_DAY = 1;
+
+const ACCESS_KEY = '3e0a35b684d94cb02fb32974d52f0a2d';
+
+const PATH_TO_FILE = path.resolve(__dirname, '..', '..', 'Database', 'db.json');
+
+const getData = async (url) => {
+  const res = await fetch(url);
+  const data = await res.json();
+  return data;
+};
+
+const getDataAndWriteIt = async () => {
```

Comment:
let's split into two methods, one for getting the data and one for writing. Please think of more conventional names